classes/02class/exercises/c02-network09/alanlima/README.md

```diff
@@ -0,0 +1,99 @@
+# c02-network09
+
+## Commands Execution Output
+
+- Commands for creating security groups for below table
+
+|Security Group Name|Rule|Type|Port Range|Source|Description
+|-|-|-|-|-|-|
+|public-sg|inbound|Custom TCP Rule|22|`<your-home-ip-address>`|SSH access from home to AWS|
+|private-sg|inbound|Custom TCP Rule|22|`<public-sg-id>`|SSH access only from public instances on AWS|
+
+```
+$ aws ec2 create-security-group \
+    --group-name public-sg \
+    --description "SSH access from home to AWS" \
+    --vpc-id vpc-027019faaf6756407
+
+GroupId: sg-035e5a3df769a4114
+
+$ aws ec2 authorize-security-group-ingress \
+    --group-id sg-035e5a3df769a4114 \
+    --cidr 122.148.135.233/32 \
+    --port 22 \
+    --protocol tcp
+
+$ aws ec2 describe-security-groups --group-ids sg-035e5a3df769a4114
+SecurityGroups:
+- Description: SSH access from home to AWS
+  GroupId: sg-035e5a3df769a4114
+  GroupName: public-sg
+  IpPermissions:
+  - FromPort: 22
+    IpProtocol: tcp
+    IpRanges:
+    - CidrIp: 122.148.135.233/32
+    Ipv6Ranges: []
+    PrefixListIds: []
+    ToPort: 22
+    UserIdGroupPairs: []
+  IpPermissionsEgress:
+  - IpProtocol: '-1'
+    IpRanges:
+    - CidrIp: 0.0.0.0/0
+    Ipv6Ranges: []
+    PrefixListIds: []
+    UserIdGroupPairs: []
+  OwnerId: '097922957316'
+  VpcId: vpc-027019faaf6756407
+
+$ aws ec2 create-security-group \
+    --group-name private-sg \
+    --description "SSH access only from public instances on AWS" \
+    --vpc-id vpc-027019faaf6756407
+GroupId: sg-0f2326bb3047f4894
+
+
+$ aws ec2 authorize-security-group-ingress \
+    --group-id sg-0f2326bb3047f4894 \
+    --source-group sg-035e5a3df769a4114 \
+    --port 22 \
+    --protocol tcp
```

Comment:
keep in mind that this will work for instances that have that SG attached. Not all instances will have that same SG, so keep in mind that depending on the case, you mind need to think in different ways to allow the connection from several instances in the public subnet.

Reply:
What would be the most used option to allow the connection from several instances? IP Range/Port or is it possible to allow AWS resources with a specific tag?